torch/onnx/utils.py

```diff
@@ -419,7 +419,13 @@ def _run_symbolic_function(g, n, inputs, aten=False):
                 # Undefined is not an ONNX operator; keep it as prim::Undefined
                 # and let the exporter handle finally eliminating these
                 return None
-
+            elif op_name == 'Loop' or op_name == 'If':
+                new_op_outputs = g.op(op_name, *inputs, outputs=n.outputsSize())
+                new_node = new_op_outputs[0].node() if n.outputsSize() > 1 else new_op_outputs.node()
+                for b in n.blocks():
+                    new_block = new_node.addBlock()
+                    torch._C._jit_pass_onnx_block(b, new_block, aten, env)
+                return new_op_outputs
```

Comment:
Here is where I would have expected the conversion into graph attributes to have happened.